pkg/controller/config-controller.go

```diff
@@ -156,6 +162,51 @@ func (r *CDIConfigReconciler) reconcileStorageClass(config *cdiv1.CDIConfig) err
 	return nil
 }
 
+func (r *CDIConfigReconciler) reconcileDefaultPodResourceRequirements(config *cdiv1.CDIConfig) error {
+	log := r.Log.WithName("CDIconfig").WithName("DefaultPodResourceRequirements")
+
+	// TODO: set as default value (0)
+	log.Info("Setting default pod resource requirements as default value (0)")
+	config.Status.DefaultPodResourceRequirements = &v1.ResourceRequirements{
+		Limits: map[v1.ResourceName]resource.Quantity{
+			v1.ResourceCPU:    *resource.NewQuantity(0, resource.DecimalSI),
+			v1.ResourceMemory: *resource.NewQuantity(0, resource.DecimalSI)},
+		Requests: map[v1.ResourceName]resource.Quantity{
+			v1.ResourceCPU:    *resource.NewQuantity(0, resource.DecimalSI),
+			v1.ResourceMemory: *resource.NewQuantity(0, resource.DecimalSI)},
+	}
+
+	if config.Spec.PodResourceRequirements != nil {
+		// TODO: Override
+		log.Info("Overriding default pod resource requirements from spec if spec is not null")
+		if config.Spec.PodResourceRequirements.Limits != nil {
+			cpu, exist := config.Spec.PodResourceRequirements.Limits[v1.ResourceCPU]
```

Comment:
Another stylistic comment, code is fine:
```go
if cpu, exist := config.Spec.PodResourceRequirements.Limits[v1.ResourceCPU]; exists {
  config.Status.DefaultPodResourceRequirements.Limits[v1.ResourceCPU] = cpu
}
```
We tend to like the if err := something; err != nil { pattern. Against its stylistic, code is fine.

The other 3 have the same.

Reply:
Thanks for your comments :) We updated commit applying this style